src/lib/tokenPrices.ts

```
@@ -14,15 +14,15 @@ export const tokenToUsd = async (token: string): Promise<number> => {
 
 	if (token === 'LOOT') {
 		priceInUsd = await getLootPrice();
+		if (!priceInUsd) return 0;
 	} else {
 		const res = await fetch(
 			`https://api.coingecko.com/api/v3/simple/price?ids=${token}&vs_currencies=usd`,
 		);
 
 		const data = await res.json();
-		if (!data) {
-			throw Error(`Unable to fetch price for ${token}`);
-		}
+
+		if (!data[token]) return 0;
```

This function should throw an exception if it fails, not a value of zero